hipparchus-core/src/main/java/org/hipparchus/fraction/BigFraction.java

```diff
@@ -267,16 +267,19 @@ public BigFraction(final double value, final double epsilon,
      * @throws MathIllegalStateException
      *             if the continued fraction failed to converge.
      */
-    public BigFraction(final double value, final int maxDenominator)
+    public BigFraction(final double value, final int maxDenominator) // TODO: long maxDenominator?
```

Comment:
I also think a long would be better here

Reply:
I adjusted it accordingly.